compatibility/deps.bzl

```diff
@@ -30,10 +30,14 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file"
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 load(
     "@daml//:deps.bzl",
+    "bazel_gazelle_sha256",
```

Comment:
why do we need this in the compat workspace now?

Reply:
The main `BUILD` file of `@daml` pulls in Gazelle, and the compatibility workspace depends on some targets defined in the top-level package, so Bazel reads that build file and fails with file not found if we don't pull it in.